source/common/upstream/subset_lb.cc

```diff
@@ -587,11 +589,15 @@ void SubsetLoadBalancer::HostSubsetImpl::update(const HostVector& hosts_added,
     }
   }
 
-  HostSetImpl::updateHosts(HostSetImpl::updateHostsParams(
-                               hosts, hosts_per_locality, healthy_hosts, healthy_hosts_per_locality,
-                               degraded_hosts, degraded_hosts_per_locality),
-                           determineLocalityWeights(*hosts_per_locality), filtered_added,
-                           filtered_removed);
+  // TODO(snowp): Right now we just pass hosts->size(), really this need to be filtering down the
```

Comment:
Turns out just passing the count around won't work since the subset lb will need a list of warmed hosts to be able to tell how many of them belong to the subset. I'll update this to use another list of hosts